web/src/integration-test/resources/ca/hendriks/Delete ingredient.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Delete ingredient
+
+  In order for recipes and user inventories to be accurate
+  As an administrator
+  I want to delete an ingredient
```

Comment:
I wonder what happens if an ingredient is used in a recipe, but the admin tries to delete the ingredient anyway?

Reply:
Do we want to do a database recipe check to see if it's in any active recipes and block/send error message if it is? 

I think the above (when amended) code will be good to start, and we can work in checks after?